File: discover_ruby/write_them_to_disc.rb
require 'httpclient'
require 'json'

headers = {
  'User-Agent' => 'Holberton_School',
  'Authorization' => 'token 81e09a5f8763afe52e37187b7e8f379e566f2dd4'
}

filename = '/tmp/14'
uri = "https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc"
target = open(filename,'w')
target.truncate(0)
newcli = HTTPClient.new

response = newcli.get_content(uri, headers)
target.write(response)
puts "The file was saved!"
target.close

File: discover_more_python/write_them_to_disc.py
import requests

response = requests.get('https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc', headers={
  'User-Agent': 'Holberton_School',
  'Authorization': 'token 3f376e3818ab2df8fcacf6cdfc2e68fbd1a63066'
})

with open('/tmp/14', 'w') as f:
    f.write(str(response.json()))

print "The file was saved!"